### ISSUE TITLE
Outdated classifier on pip

### ISSUE THREAD

**Issue by ricardovsilva** [bhrigu123/classifier#77]
Pip version of classifier is outdated.
For example, pip version does not create settings file.

**Comment by bhrigu123** [comment]
Yes, the package isn't updated on PyPi (pip) yet. Some minor changes are left.
Should be done in about a week or two.
